uninstaller.nsh

```diff
@@ -1,6 +1,13 @@
 !macro customUnInstall
-    ExecWait '"C:\Program Files\remoteit-bin\remoteit.exe" -j service uninstall'
-    RMDir /r "$APPDATA\remoteit"
+    MessageBox MB_YESNO "Would you like to unregistered your device?" IDYES true IDNO false
+    true:
+        ExecWait '"C:\Program Files\remoteit-bin\remoteit.exe" -j service uninstall'
+        RMDir /r "$APPDATA\remoteit"
+        Goto next
+    false:
+        RMDir /r "$APPDATA\remoteit\log"
+        Goto next
```

Comment:
```suggestion
    MessageBox MB_YESNO "Would you like to unregistered your device?" IDYES true IDNO false
    true:
        ExecWait '"C:\Program Files\remoteit-bin\remoteit.exe" -j uninstall'
        RMDir /r "$APPDATA\remoteit"
        Goto next
    false:
        ExecWait '"C:\Program Files\remoteit-bin\remoteit.exe" -j service uninstall'
        RMDir /r "$APPDATA\remoteit\log"
        Goto next
```
This is great thank you @mauricio0000 ! Just one little edit we need for cli. 👍

Reply:
ready